R/EpiStats.R

```diff
@@ -13,6 +13,11 @@
 #'        (35, 45], (45, 55], (55, 65], (65, 100].
 #' @param race Whether to stratify by racial status. Default is TRUE.
 #' @param browser Run function in interactive browser mode. Default is FALSE.
+#' @param init.hiv.prev Initial HIV prevalence of estimated model, vector of size
+#'.        pertaining to prevalence among three racial classes (black, hispanic and
+#'         white respectively). If "NULL", ARTnet will handle calculation of
+#'         prevalence through ARTnet data. if `race = FALSE`, will take first element
```

Comment:
NULL isn't a character vector. As previously discussed, all code in the documentation should be wrapped in `\code{x]`.